runtime/hydradx/src/tests.rs

```diff
@@ -363,3 +367,24 @@ mod xcm_fee_payment_api_tests {
 		});
 	}
 }
+
+#[test]
+fn assert_kusama_root_account() {
```

Comment:
wondering if we could come up with the concrete use case in the test to justify the need for adding this `GlobalConsensusConverter`?

By doing so we dont need to couple our test to implementation details `GlobalConsensusConverter`, so it will become more flexible.

Reply:
Good suggestion but that would require a complex integration test that would mock P<>K bridge and performing a transact instruction execution remotely. I would keep it for later as now it is a pretty straight forward change inherited from polkadot sdk and we need the change to be merged asap